### PR TITLE
Add payment transfer test for Scalar DL

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-3410

~It needs the latest client SDK which hasn't been released yet.~
~Unit tests should fail because the public SDK is old for now.~

~I will update the client SDK version after the latest client SDK is released.~

### CODE REVIEW

File: scalardl/contract/Create.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+
+public class Create extends Contract {
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> property) {
+    if (!(argument.containsKey("id") && argument.containsKey("balance"))) {
+      throw new ContractContextException("required key 'id' and 'balance' is missing");
```
Comment (siyopao) [inline scalardl/contract/Create.java]:
```suggestion
      throw new ContractContextException("required key 'id' or 'balance' is missing");
```

File: scalardl/contract/Transfer.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import javax.json.Json;
+import javax.json.JsonObject;
+
+public class Transfer extends Contract {
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> property) {
+    if (!(argument.containsKey("from")
+        && argument.containsKey("to")
+        && argument.containsKey("amount"))) {
+      throw new ContractContextException("required keys 'from', 'to' and 'amount' is missing");
```
Comment (siyopao) [inline scalardl/contract/Transfer.java]:
```suggestion
      throw new ContractContextException("required key 'from', 'to' or 'amount' is missing");
```

Comment (yito88) [reply scalardl/contract/Transfer.java]: Thanks! Done!

File: scalardl/contract/Create.java (hunk 1 above)
Comment (yito88) [reply scalardl/contract/Create.java]: Thanks! Done!

**Comment by yito88** [comment]
I've updated SDK and supported commit checking for unknown failure.
I found and fixed bugs the commit checking failed in my local environment.
Please review again!

**Pull request merged**